<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="5TS206424426L7" name="com.nbcuni.test.nbcidx.member.link.errorcodes.TC3374_MemberLogin_Error_502" tests="1" failures="0" timestamp="10 Sep 2014 17:57:39 GMT" time="7.967" errors="1">
  <testcase name="memberLogin_error502" time="7.967" classname="com.nbcuni.test.nbcidx.member.link.errorcodes.TC3374_MemberLogin_Error_502">
    <error type="java.lang.AssertionError" message="Failed : HTTP error code : 400">
      <![CDATA[java.lang.AssertionError: Failed : HTTP error code : 400
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at com.nbcuni.test.nbcidx.member.link.errorcodes.TC3374_MemberLogin_Error_502.memberLogin_error502(TC3374_MemberLogin_Error_502.java:128)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- memberLogin_error502 -->
</testsuite> <!-- com.nbcuni.test.nbcidx.member.link.errorcodes.TC3374_MemberLogin_Error_502 -->
